tavern/config.go

```diff
@@ -251,27 +257,27 @@ func (cfg *Config) NewGitImporter(client *ent.Client) *tomes.GitImporter {
 
 // IsDefaultTomeImportEnabled returns true default tomes should be imported.
 func (cfg *Config) IsDefaultTomeImportEnabled() bool {
```

Comment:
we can probs remove this functions now right?

Reply:
No this is used by tavern rn